src/pages/en/core-concepts/framework-components.md

```diff
@@ -78,41 +78,9 @@ the user scrolls down and the component is visible on the page -->
 
 ### Available Hydration Directives
 
-#### `client:load`
+There are serveral hydration directives available for UI framework components, such as `client:load`, `client:idle`, and more!
```

Comment:
```suggestion
There are serveral hydration directives available for UI framework components: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` and `client:only=" "`
```

Can we maybe meet in the middle, and since there are only 5 of them, list them all here??  ;)